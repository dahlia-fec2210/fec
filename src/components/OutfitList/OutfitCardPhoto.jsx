import React from 'react';

function OutfitCardPhoto({ photo, clothingPiece }) {
  return (
    <img className="related-outfit-photo" src={photo || 'https://img.ltwebstatic.com/images3_pi/2022/04/06/16492430704a5786a3329d6838490cfcc903aa6996_thumbnail_600x.webp'} alt={clothingPiece} />
  );
}

export default OutfitCardPhoto;
