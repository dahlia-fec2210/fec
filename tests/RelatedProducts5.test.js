/* eslint-disable no-undef */
/* eslint-disable import/extensions */
/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import userEvent from '@testing-library/user-event';
import renderer from 'react-test-renderer';

import OutfitCardPhoto from '../src/components/OutfitList/OutfitCardPhoto.jsx';

test('renders photo for outfit list card with product 37315', async () => {
  const tree = renderer
    .create(<OutfitCardPhoto photo="https://img.ltwebstatic.com/images3_pi/2022/04/06/16492430704a5786a3329d6838490cfcc903aa6996_thumbnail_600x.webp" clothingPiece={37315} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
