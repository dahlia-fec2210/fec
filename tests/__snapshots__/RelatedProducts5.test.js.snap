// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders photo for outfit list card with product 37315 1`] = `
<img
  alt={37315}
  className="related-outfit-photo"
  src="https://img.ltwebstatic.com/images3_pi/2022/04/06/16492430704a5786a3329d6838490cfcc903aa6996_thumbnail_600x.webp"
/>
`;
